.github/ISSUE_TEMPLATE/patch_release.md

```diff
@@ -0,0 +1,71 @@
+---
+name: Publish a patch release
+about: Publish a new Orchard Core patch release
+title: 'Release v'
+labels: release
+assignees: ''
+
+---
+
+# Release Patch Preparation Guide
+
+## Step 1: Backporting Pull Requests
+
+1. **Identify PRs**: Review any pull requests (PRs) that need to be backported to the release branch.
+2. **Backporting Process**: For PRs merged into the main branch that need to be applied to `release/2.1`, comment on the merged PR with `/backport to release/2.1`. This action will trigger GitHub to create a new PR with the same changes for the `release/2.1` branch.
+3. **Merge PRs**: Once all necessary PRs are created, merge them into the `release/2.1` branch.
+
+## Step 2: Administration Tasks
+
+#### Create Pull Request:
+
+   - From the release branch (e.g., `release/2.1`), create a new branch for your release (e.g., `release-notes/2.1.1`).
+   - Update version references in the documentation. Refer to [this PR](https://github.com/OrchardCMS/OrchardCore/pull/17065/files) for an example.
+   - Version Updates Checklist:
+     - [ ] **Update `OrchardCore.Commons.props`**: Set `<VersionSuffix></VersionSuffix>` to prevent preview build numbers. Ensure `VersionPrefix` matches the released version.
+     - [ ] **Update Module Versions**: Modify `src/OrchardCore/OrchardCore.Abstractions/Modules/Manifest/ManifestConstants.cs` as needed.
+     - [ ] **Release Notes**: Finalize release notes in the documentation, including:
+       - Highlights and goals of the release.
+       - Prerequisites for running the new version.
+       - Upgrade steps and any breaking changes.
+     - [ ] **Update Documentation Navigation**: Add the release notes page to `mkdocs.yml` navigation and remove it from `not_in_nav`.
+     - [ ] **Version Mentions**: Update all references to the new version throughout the documentation, including:
+       - [Status in the root README](https://docs.orchardcore.net/en/latest/#status)
+       - CLI templates and commands.
+       - Relevant guides, such as the [Creating a new decoupled CMS Website](https://docs.orchardcore.net/en/latest/guides/decoupled-cms/) guide.
```

Comment:
It would be better to have the same in the other template too.

Reply:
done